app-office/moneydance/moneydance-2019.3.1880.ebuild

```diff
@@ -41,9 +42,15 @@ src_compile() {
 
 src_install() {
 	java-pkg_dojar lib/*.jar
-	java-pkg_dolauncher moneydance --main "Moneydance" --java_args "-client -Dawt.useSystemAAFontSettings=gasp -Dawt.useSystemAAFontSettings=on -Xmx1024m"
+	# java-pkg_dolauncher moneydance --main "Moneydance" --java_args "-client -Dawt.useSystemAAFontSettings=gasp -Dawt.useSystemAAFontSettings=on -Xmx1024m"
```

Comment:
Don't leave stuff commented out.

Reply:
I left it intentionally, when the ```virtual/jdk``` will be unmasked. But okay, removed.